Defanged IP Address

ErzaTitania-2001 opened the pull request:
<!--
Thank you for sending the PR! We appreciate you spending the time to work on these changes.

Help us understand your motivation by explaining why you decided to make this change.
You can learn more about contributing to NeoAlgo here: https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md

Happy Contributing!

-->

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description

yes

### Checklist

- [y ] I've read the contribution guidelines.
- [y ] I've checked the issue list before deciding what to submit.
- [y ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Defanged IP Address #1750 as assigned by @iamrajiv

ErzaTitania-2001 committed:
Defanged IP Address #1750

ErzaTitania-2001 committed:
Rename GC3.java to Defanged_IPv4.java

ErzaTitania-2001 committed:
Update Defanged_IPv4.java

ErzaTitania-2001 committed:
Update README.md

ErzaTitania-2001 commented:
@iamrajiv. Defanged IP Address #1750

atarax665 commented on Java/cp/Defanged_IPv4.java:
```diff
@@ -0,0 +1,39 @@
+/* Given a valid (IPv4) IP address, return a defanged version of of that IP address. A defanged IP address replaces every period "." with "[.]".
+
+Input : address : "1.1.1.1"
```
Remove the  sample I/O from here.

ErzaTitania-2001 replied:
Done @atarax665

atarax665 commented on Java/cp/Defanged_IPv4.java:
```diff
@@ -0,0 +1,39 @@
+/* Given a valid (IPv4) IP address, return a defanged version of of that IP address. A defanged IP address replaces every period "." with "[.]".
```
```suggestion
/* Given a (IPv4) IP address, return a defanged version of of that IP address. A defanged IP address replaces every period "." with "[.]".
```
I have edited the problem description because if the given IP address is always valid then the validation part of the code would become redundant.

ErzaTitania-2001 replied:
Done @atarax665

atarax665 commented on Java/cp/Defanged_IPv4.java:
```diff
@@ -0,0 +1,39 @@
+Defanged IPv4 address : 2500[.]100[.]50[.]0
+Enter IPv4 address
+1.1.1.1.
+Not a valid IPv4 address
```
Add a blank line after Sample I/O.

ErzaTitania-2001 replied:
Done @atarax665

atarax665 reviewed (changes requested):
LGTM, do the suggested changes. :+1:

ErzaTitania-2001 committed:
Update Defanged_IPv4.java

Added blank line after sample input and out.
Removed sample input and output comment line.

iamrajiv committed:
formatted
